Feat/workspace external members (#493)
BLSQ/openhexa-app: hexa/workspaces/templates/workspaces/mails/invite_external_user.txt

You have been invited to join the workspace {{workspace}} by {{owner}}.

> Use the following link proceed {{ workspace_signup_url }}

Reviewer: ```suggestion
Create your account using this link: {{ workspace_signup_url }}
```

> 

Reviewer: Add a blank line
